qiskit/visualization/circuit/circuit_visualization.py

```diff
@@ -0,0 +1,663 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2018.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Module for the primary interface to the circuit drawers.
+
+This module contains the end user facing API for drawing quantum circuits.
+There are 3 available drawer backends:
+
+ 0. ASCII art
+ 1. LaTeX
+ 2. Matplotlib
+
+This provides a single function entry point to drawing a circuit object with
+any of the backends.
+"""
+
+import logging
+import os
+import subprocess
+import tempfile
+from warnings import warn
+
+from qiskit import user_config
+from qiskit.utils import optionals as _optionals
+from . import latex as _latex
+from . import text as _text
+from . import matplotlib as _matplotlib
+from . import circuit_utils
+from .. import utils
```

Comment:
Just nit-picking, but it'd be a shade nicer to `from ..utils import _trim as trim_image` (or something) just so the reader of the code at the usage site doesn't need to think about the difference between `circuit_utils` and `utils` (especially if we rename `circuit_utils` to `_utils`).

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/8306/commits/64844c9c120734626e9744569323ac521f64099a.